Source/WebCore/platform/graphics/avfoundation/InbandTextTrackPrivateAVF.cpp

```diff
@@ -488,19 +488,89 @@ void InbandTextTrackPrivateAVF::resetCueValues()
 
 void InbandTextTrackPrivateAVF::setMode(InbandTextTrackPrivate::Mode newMode)
 {
-    if (!m_owner)
-        return;
-
     InbandTextTrackPrivate::Mode oldMode = mode();
     InbandTextTrackPrivate::setMode(newMode);
 
-    if (oldMode == newMode)
+    if (m_owner && oldMode != newMode)
+        m_owner->trackModeChanged();
+}
+
+bool InbandTextTrackPrivateAVF::processVTTFileHeader(CMFormatDescriptionRef formatDescription)
+{
+    using namespace PAL;
```

Comment:
probably not needed as the only call to CM method is already prefixed by `PAL::`